src/mpid/ch3/channels/nemesis/include/mpid_nem_datatypes.h

```diff
@@ -63,7 +63,7 @@
    For optimization, we want the cell to start at a cacheline boundary
    and the cell length to be a multiple of cacheline size.  This will
    avoid false sharing.  We also want payload to start at an 8-byte
-   boundary to to optimize memcpys and dataloop operations on the
+   boundary to to optimize memcpys and datatype operations on the
```

Comment:
This needs to be separated into a different commit.  I'll take care of it in the next round.

Reply:
Fixed.